ersilia/core/tracking.py

```diff
@@ -536,7 +537,27 @@ def track(self, input, result, meta):
         self.docker_client = SimpleDocker()
         json_dict = {}
         input_data = read_csv(input)
-        result_data = read_csv(result)
+        # Create a temporary file to store the result if it is a generator
```

Comment:
Reword to "create a temporary file to store the result if..."

Reply:
I am not sure i got you well. What you want me to reword it to looks the same to me
"create a temporary file to store the result if it is a generator"

Reply:
Oh my bad, I think I got confused a bit when I read the word output at L548. I think this is okay, let me mark this conversation as resolved and merge this.